src/wp-admin/includes/media.php

```diff
@@ -3843,3 +3843,12 @@ function wp_media_attach_action( $parent_id, $action = 'attach' ) {
 		exit;
 	}
 }
+
+/**
+ * Returns the default image output mapping.
+ *
+ * @since 6.1.0
+ */
+function wp_default_image_output_mapping() {
+	return array( 'image/jpeg' => 'image/webp' );
+}
```

Comment:
This implementation is problematic as it would override any other mappings present. I think you will need to also include the actual filter parameter `$output_format` in this callback function and then set `$output_format['image/jpeg'] = 'image/webp'` rather than overriding the entire array.

Would be good to add a test to ensure this behavior, e.g. test that whatever other random mapping that is filtered in there (e.g. for another MIME type than `image/jpeg`) still remains in there after running the callback.

Reply:
good point

Reply:
adjusted to use passed value and only set mapping for jpeg